app/helpers/repository_helper.rb

```diff
@@ -90,7 +90,7 @@ def add_files(files, user, repo, path: '/', txn: nil, check_size: true, required
   # nil, the boolean indicates whether any errors were encountered which mean the caller should not
   # commit the transaction. The array contains any error or warning messages as arrays of arguments
   # (each of which can be passed directly to flash_message from a controller).
-  def remove_files(files, user, repo, path: '/', txn: nil)
+  def remove_files(files, user, repo, path: '/', txn: nil, option: 'File')
```

Comment:
Usually with this sort of parameter that can either be one thing or another (in this case 'file' or 'directory') it is better to use a boolean value and name the parameter something less generic. "option" doesn't clearly describe what we're using this parameter for. What are some other names we could use?